### PR TITLE
#8 retrieve all positions and show as list

### PR DESCRIPTION
**Pull request by Marlin-Noethig**
fully coverd ticket #8 
implemented the full logic to retrieve and show data from the db at the frontend

### CODE REVIEW

File: frontend/src/pages/BudgetOverview.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import usePositions from "../hooks/usePositions";
+import PositionList from "../components/budget-overview/PositionList";
+
+export default function BudgetOverview(){
+
+    const positions = usePositions();
+
+    console.log(positions);
```
Comment (PascalWo) [inline frontend/src/pages/BudgetOverview.tsx:8]: maybe delete console.log for PR :)

**Review by PascalWo — APPROVED** [review]
great work :)

File: frontend/src/pages/BudgetOverview.tsx (hunk 1 above)
Comment (Marlin-Noethig) [reply frontend/src/pages/BudgetOverview.tsx:8]: arrr damn, i forgot. Good eyes!

Comment (Marlin-Noethig) [inline frontend/src/pages/BudgetOverview.tsx:8]:
```suggestion
```

Comment (Marlin-Noethig) [inline frontend/src/pages/BudgetOverview.tsx:8]:
```suggestion
```

File: backend/src/test/java/de/neuefische/backend/controller/PositionsControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    @BeforeEach
+    public void setUp(){
+        positionsRepo.deleteAll();
+    }
+
+    @Test
+    void getPositions() {
+        //will be written, when post-method is implemented to controller
+        assertTrue(true);
```
Comment (Flooooooooooorian) [inline backend/src/test/java/de/neuefische/backend/controller/PositionsControllerTest.java:33]: i dont like this. Why not save an entity in your repo an make your get request independetly from the post?

**Pull request merged**